### PR TITLE
An option for use_svd for timing model.

### PR DESCRIPTION
**Pull request by JacobCT**

### CODE REVIEW

File: enterprise_warp/results.py [hunk 1]
```diff
@@ -405,7 +409,7 @@ def get_psr_dirs(self):
     psr_dir_mask = [check_if_psr_dir(dd) for dd in out_subdirs]
     self.psr_dirs = out_subdirs[psr_dir_mask]
     if self.psr_dirs.size == 0:
-      self.psr_dirs = np.array([''])
+       self.psr_dirs = np.array([''])
```
Comment (bvgoncharov) [inline enterprise_warp/results.py]: It looks like this extra indent is not needed

File: enterprise_warp/results.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 from corner import corner
 from datetime import datetime
 from bilby import result as br
-from chainconsumer import ChainConsumer, Chain, PlotConfig
+#from chainconsumer import ChainConsumer, Chain, PlotConfig
```
Comment (bvgoncharov) [inline enterprise_warp/results.py]: It would be a good idea to uncomment this line for a general user to be able to use ChainConsumer

**Review by bvgoncharov — CHANGES_REQUESTED** [review]

**Review by bvgoncharov — APPROVED** [review]
Looks good!

File: enterprise_warp/results.py (hunk 1 above)
Comment (JacobCT) [reply enterprise_warp/results.py]: Should be resolved now.

File: enterprise_warp/results.py (hunk 2 above)
Comment (JacobCT) [reply enterprise_warp/results.py]: Should be resolved now.

**Pull request merged**